app/frontend/src/components/NotificationBar/_NotificationBar_Close.scss

```diff
@@ -0,0 +1,97 @@
+// External Imports
+@use "sass:map";
+
+// Internal Imports
+@use "../Basics" as *;
+
+/*****************************
+*** Notification Bar class ***
+*****************************/
+$themes: "primary", "success", "warning", "danger";
+
+p {
+  padding: 0;
+  margin: 0;
+}
+
+.notification-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1em;
+}
+
+.notification-bar-close {
+  cursor: pointer;
+  border: none;
+  background: none;
+  margin-right: auto;
+}
+
+.notification-bar-fade-appear {
+  opacity: 0;
+}
+
+.notification-bar-fade-appear-active {
+  opacity: 1;
+  transition-duration: 300ms;
+}
+
+.notification-bar-fade-enter {
+  opacity: 0;
+}
+
+.notification-bar-fade-enter-active {
+  opacity: 1;
+  transition-duration: 300ms;
+}
+
+.notification-bar-fade-exit {
+  opacity: 1;
+}
+
+.notification-bar-fade-exit-active {
+  opacity: 0;
+  transition-duration: 300ms;
+}
+
+.text {
+  text-align: center;
+  margin-bottom: 8px;
+}
+
+/*********************************************
+*** Notification Bar classes concerning color themes ***
+*********************************************/
+$theme-values: (
+  "primary": (
+    "background-color": #8abbff,
+    "color": black
+  ),
+  "success": (
+    "background-color": #72ca9b,
+    "color": black
+  ),
+  "warning": (
+    "background-color": $color-tan,
+    "color": black
+  ),
+  "danger": (
+    "background-color": #fa999c,
+    "color": black
```

Comment:
This is fine overall, but we don't really want to hardcode colors in case we do theming in the future. Let's set all these to colors we already have, like color-tan, and leave a comment that we will set these to different values in the future. Thoughts?